### DISCUSSION TITLE
Fallbacking and runtime only build

### DISCUSSION THREAD

**Discussion by pau-santesmasses** [intlify/bundle-tools#55] [Q&A] [answered]
Should it be possible to use `fallbackFormat: true` in `createI18n()` and still use the runtime only build?

With the settings above, the build is successful but it fails to run,  with `TypeError: o is not a function.`

It works fine with `runtimeOnly: false`, but I was trying to see if I could optimize the bundle.

Thanks in advance!

**Comment by kazupon** [comment, marked as answer]
It could be a bug in vue-i18n-next.
https://github.com/intlify/vue-i18n-next/issues/768

It has already been fixed in vue-i18n v9.2.0-beta.19.

**Reply by pau-santesmasses** [reply]
Amazing. Thank you!
